boilerplate/update

```diff
@@ -125,7 +125,8 @@ while read directory junk; do
   fi
   echo "adding makefile includes."
   for inc in $(find "${DIR}/${directory}" -type f -name '*.mk' | sort); do
-    echo "include ${inc#$REPO_ROOT/}" >> "${NEXUS_MK}"
+    real_include=`realpath $inc`
```

Comment:
`git remote get-url` returns a path with resolved symlinks, so this is failing in case your `TMPDIR` contains a symlink

Reply:
I don't understand the symlink thing.

Reply:
Or perhaps it was the `git remote get-url` thing that was throwing me off. That's not in play here. Perhaps you mean `find` is not resolving symlinks by default? If that's the issue, we should use `find -L` rather than trying to hack it up after the fact.

Reply:
Summarizing offline explanation:

When we calculate `$REPO_ROOT` on [L64](https://github.com/openshift/boilerplate/pull/32/files#diff-fb7297e9f79b9a50001fa66af778f797R64), `git rev-parse` returns the result fully resolved. But `$DIR` is calculated using `pwd` ([L17](https://github.com/openshift/boilerplate/pull/32/files#diff-fb7297e9f79b9a50001fa66af778f797R17)) which doesn't fully resolve. So if the path contains symlinks, the remove-left-pattern below won't match.

Notwithstanding the weirdness of @bdematte's system having a symlink from `/var` to `private/var` (/me rolls eyes at Mac *again*) I _guess_ this makes it important that both are fully resolved.

Arguably it would be better to do it at L17 instead, e.g.

```bash
DIR=$(realpath ${0%/*})
```